mithril-test-lab/mithril-devnet/devnet-mkfiles.sh

```diff
@@ -606,7 +606,7 @@ done
     cat >> activate.sh <<EOF
 echo ">> Wait for Cardano pools to be activated"
 POOLS_ACTIVATION_WAIT_ROUND_DELAY=2
-POOLS_ACTIVATION_WAIT_ROUNDS_MAX=\$(echo "scale=0; 5 * $EPOCH_LENGTH * $SLOT_LENGTH / \$POOLS_ACTIVATION_WAIT_ROUND_DELAY" | bc)
+POOLS_ACTIVATION_WAIT_ROUNDS_MAX=\$(echo "scale=0; 20 * $EPOCH_LENGTH * $SLOT_LENGTH / \$POOLS_ACTIVATION_WAIT_ROUND_DELAY" | bc)
```

Comment:
what does that change stand for?

Reply:
This control the max number of attempt that's the script do when starting a node, why use a formula ? Because the network startup time change based on the network parameters, a fast network would start faster than a slow network, so it's difficult to use the same value for everyone.
The problem here was that on my machine this computation lead to a low value of 9, most of the time it was not enough for the network to start (it takes 10 or 12 attempts).
By quadrupling the base factor this should give enough room for everyone for the network to start at the cost of waiting longer in case of a failure.